PendingReleaseNotes

```diff
@@ -17,6 +17,13 @@
   the balancer was included in the ``always_on_modules`` list, but needed to be
   turned on explicitly using the ``ceph balancer on`` command.
 
+* MGR: the "cloud" mode support for ``device_failure_prediction_mode`` has been
+  removed. Also, the manager module of ``ceph-mgr-diskprediction-cloud`` has
+  been dropped as well. Because the service offered by https://www.diskprophet.com/
+  is not accessible anymore and we don't have any replacement for it so far,
+  we have to remove this feature. But the "local" mode of
+  ``device_failure_prediction_mode`` is still supported.
+
```

Comment:
How about the following? the url is not accessible so let's skip it

```suggestion
* MGR: the "cloud" mode of the diskprediction module is not supported anymore and the ``ceph-mgr-diskprediction-cloud`` manager module has been removed. This is because the external cloud service run by ProphetStor is no longer accessible and there is no immediate replacement for it at this time. The "local" prediction mode will continue to be supported.

```

Reply:
@neha-ojha much better! thanks! i just incorporated your change and repushed.